metadatastore/commands.py

```diff
@@ -1165,3 +1165,46 @@ def find_last(num=1):
         if next(c) == num:
             raise StopIteration
         yield _cache_run_start(rs)
+
+
+def get_header(run_start, allow_no_run_stop):
```

Comment:
The plan is to re-name the DP headers to 'RunRecord' or something like that.  This dict is more along the lines of the original thing we meant by 'header'.